var/spack/repos/builtin/packages/netlib-scalapack/package.py

```diff
@@ -48,10 +50,22 @@ class NetlibScalapack(Package):
 
     def install(self, spec, prefix):
         options = [
-            "-DBUILD_SHARED_LIBS:BOOL=%s" % ('ON' if '+shared' in spec else 'OFF'),
-            "-DBUILD_STATIC_LIBS:BOOL=%s" % ('OFF' if '+shared' in spec else 'ON'),
-            "-DUSE_OPTIMIZED_LAPACK_BLAS:BOOL=ON", # forces scalapack to use find_package(LAPACK)
-            ]
+            "-DBUILD_SHARED_LIBS:BOOL=%s" % ('ON' if '+shared' in spec else
+                                             'OFF'),
+            "-DBUILD_STATIC_LIBS:BOOL=%s" % ('OFF' if '+shared' in spec else
+                                             'ON'),
+            # forces scalapack to use find_package(LAPACK):
+            "-DUSE_OPTIMIZED_LAPACK_BLAS:BOOL=ON",
+        ]
+
+        # Make sure we use Spack's Lapack:
+        options.extend([
+            '-DLAPACK_FOUND=true',
```

Comment:
unfortunately this has to be done here because otherwise Lapack will be picked up from anywhere including system provided lapack irrespectively of what Spack is using